### ISSUE TITLE
Nios2 and sdram

### ISSUE THREAD

**Issue by thegabriele97** [jvitkauskas/Altera-Cyclone-IV-board-V3.0#2]
Hi,
This is the only repository I found online that mention this board (I don't know why it's so unknown!). 

I would like to ask you if you ever tried to use the sdram chip with nios 2 soft processor. I try and I can't make it works! :(

**Comment by jvitkauskas** [comment]
No, I've actually got that board only to play around and tried doing only really basic things.

**Comment by alainlou** [comment]
hey @thegabriele97, it's been a while but if you're trying to run a soft processor on the board I recently added some support for this board with the litex toolchain!

https://github.com/litex-hub/litex-boards
